lib/scope.js

```diff
@@ -110,15 +114,11 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
     var in_destructuring = null;
     var in_export;
     var tw = new TreeWalker(function(node, descend){
-        var create_a_block_scope =
-            (options.screw_ie8 && node instanceof AST_Catch) ||
```

Comment:
I don't think there are any issues on master. But backporting any new catch scope tests there would be a good idea.